imagersite/imager_profile/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.encoding import python_2_unicode_compatible
+
+
+class ActiveProfileManager(models.Manager):
+    def get_queryset(self):
+        return super(ActiveProfileManager, self).get_queryset()\
+            .filter(user__is_active=True)
+
+
+@python_2_unicode_compatible
+class ImagerProfile(models.Model):
+    PHOTO_TYPES = (
+        ('N', 'Nature Photography'),
+        ('P', 'Paranormal Photography'),
+        ('A', 'Aerial Photography'),
+        ('B', 'Black and White Photography'),
+        ('F', 'Forensic Photography')
+    )
+    user = models.OneToOneField(
+        User,
+        related_name="profile",
+        null=False
+    )
+    fav_camera = models.CharField(max_length=256)
+    address = models.TextField()
+    url = models.URLField()
```

Comment:
remember that the default for Django model fields is to be NOT NULL.  This means that without specifying that the address and URL fields here can be empty, you will be required to provide these values.  Is that what you want?